install/kubernetes/templates/istio-pilot.yaml.tmpl

```diff
@@ -90,7 +90,7 @@ spec:
       - name: discovery
         image: {PILOT_HUB}/pilot:{PILOT_TAG}
         imagePullPolicy: IfNotPresent
-        args: ["discovery", "-v", "2", "--admission-service", "istio-pilot"]
+        args: ["discovery", "-v", "2", "--port", "8080", "--admission-service", "istio-pilot"]
```

Comment:
Can we leave it as default port ? This is the 'insecure' port, that envoy forwards to, I don't see a good reason to change the default and then set a port to put it back to the old value.